plug.gemspec

```diff
@@ -16,11 +16,12 @@ Gem::Specification.new do |s|
 
   s.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.md']
 
-  s.add_dependency 'rails', '<= 5.1.4'
+  s.add_dependency 'rails', '<= 5.1.6'
```

Comment:
we could get rid of the versioning, or loosen it up so that we don't have to constantly update it.

Reply:
👍